corehq/apps/locations/migrations/0013_get_stock_levels_from_config.py

```diff
@@ -0,0 +1,78 @@
+# encoding: utf-8
+from south.v2 import DataMigration
+from dimagi.utils.couch.database import get_db
+from corehq.apps.domain.models import Domain
+from corehq.apps.locations.models import LocationType
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        """call the save method on all relevant location types"""
+        db = get_db()
+        for domain in Domain.get_all():
+            if domain.commtrack_enabled:
```

Comment:
Good call.  Are you concerned about the performance of this?
```python
for location_type in LocationType.objects.iterator():
    location_type.save()
```
There are 998 location type objects currently in the prod db.